### PR TITLE
chore(suite-desktop): update electron, electron-builder and esbuild

### PR DESCRIPTION
**Pull request by keraf**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -9499,7 +9668,7 @@ electron-updater@^4.3.5:
     lodash.isequal "^4.5.0"
     semver "^7.3.2"
 
-electron@*, electron@^11.2.1:
+electron@*:
```
Comment (prusnak) [inline yarn.lock]: `electron@*:` still points to 11.2.1 - is that intended?

Comment (keraf) [reply yarn.lock]: Good catch. I don't think it should be as nothing depends on it, I'll dig into that one.

Comment (keraf) [reply yarn.lock]: 887d0ae

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "repository": "https://github.com/trezor/trezor-suite.git",
     "license": "SEE LICENSE IN LICENSE.md",
     "engines": {
-        "node": ">=12",
+        "node": ">=14",
```
Comment (tomexx) [inline package.json]: I'd keep minor and fix version as in build target `>=14.16.0`

**Review by szymonlesisz — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (keraf) [reply package.json]: 5166fc4

**Pull request merged**